mcmd/commands/add.py

```diff
@@ -72,6 +72,29 @@ def arguments(subparsers):
                              type=str,
                              help="The token")
 
+    p_add_theme = p_add_subparsers.add_parser('theme',
+                                              help='Upload a bootstrap theme')
+    p_add_theme.set_defaults(func=add_theme,
+                             write_to_history=True)
+    p_add_theme.add_argument('bs3',
+                             type=str,
+                             metavar='Bootstrap 3 theme',
+                             help="The bootstrap 3 css theme file")
+    p_add_theme.add_argument('--bs4',
+                             type=str,
+                             metavar='Bootstrap 4 theme',
```

Comment:
As discussed we will use the `metavar` only for optional arguments that don't look nice without it, for example:

```
--from-issue FROM_ISSUE  <-- without metavar
--from-issue NUMBER      <-- with metavar
```